src/Behat/Behat/Gherkin/Specification/LazyFeatureIterator.php

```diff
@@ -62,7 +62,7 @@ final class LazyFeatureIterator implements SpecificationIterator
     /**
      * Initializes specifications.
      *
-     * @param string[]          $paths
+     * @param string[]                 $paths
      * @param FeatureFilterInterface[] $filters
```

Comment:
Personally, I would **not** align variable names in PHPDoc `@param` tags - I think that just causes unhelpful diffs if params are added/removed or the type of one param is renamed / changed. Same reasons as not aligning array values / method param names, basically.